packages/js/result/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@polywrap/result",
```

Comment:
No, the result package is useful on it's own even without the core. It could be replaced by another third party if we wished (there is a similar one which this one is based of off)